generated/nidigital/session.py

```diff
@@ -2676,6 +2676,85 @@ def export_signal(self, signal, signal_identifier, output_terminal):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, timeout):
+        # This is slightly modified codegen from the function
+        vi_ctype = _visatype.ViSession(self._vi)  # case S110
+        site_list_ctype = ctypes.create_string_buffer(site_list.encode(self._encoding))  # case C020
+        waveform_name_ctype = ctypes.create_string_buffer(waveform_name.encode(self._encoding))  # case C020
+        samples_to_read_ctype = _visatype.ViInt32(samples_to_read)  # case S150
+        timeout_ctype = _visatype.ViReal64(timeout)  # case S150
```

Comment:
If this was codegenerated, the timeout conversion would go here. I think we should make this as close to what the codegen would spit out as possible.

We should possibly open an issue on the generator's limitation and reference it here. We may not fix it but at least we have the opportunity to discuss it so the reasoning is clear.

Reply:
Issue #1114 created

Template updated to move the conversion to the "code generated" part.